ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -238,9 +239,13 @@ public String toString(final FormatOptions formatOptions) {
   }
 
   private Optional<Column> findColumnMatching(final Predicate<Column> predicate) {
+    // At the moment, it's possible for some column names to have multiple matches, e.g.
```

Comment:
Note: this is temporary and will go away once we're not copying key columns into the value and we ditch meta columns.